src/api/GraphQLApi.js

```diff
@@ -1,52 +1,17 @@
-const gql = require("graphql-tag");
+const { getQuestionnaire } = require("./queries");
 const fetch = require("node-fetch");
+const gql = require("graphql-tag");
 
 global.fetch = fetch;
 
 class GraphQLApi {
-  constructor(client) {
-    this.client = client;
+  constructor(apolloClient) {
+    this.apolloClient = apolloClient;
   }
 
   getAuthorData(questionnaireId) {
-    return this.client.query({
-      query: gql`
-        query GetQuestionnaire($questionnaireId: Int!) {
-          questionnaire(id: $questionnaireId) {
-            id
-            title
-            description
-            theme
-            legalBasis
-            navigation
-            sections {
-              id
-              title
-              description
-              pages {
-                ... on QuestionPage {
-                  id
-                  title
-                  description
-                  guidance
-                  pageType
-                  type
-                  mandatory
-                  answers {
-                    id
-                    description
-                    guidance
-                    qCode
-                    label
-                    type
-                    mandatory
-                  }
-                }
-              }
-            }
-          }
-        }
-      `,
+    return this.apolloClient.query({
+      query: gql(getQuestionnaire),
```

Comment:
Good spot, forgot about that. The reason it was done at source was because MockGraphQLApi.js used to pass the query directly to the mockServer implementation, which only accepted String GraphQL queries. Since I've refactored this to mock the network interface it's no longer applicable. Also, the best practice way of converting the GraphQL AST to string is using the print method as is being done [here](https://github.com/ONSdigital/eq-publisher/pull/13/files#diff-fb06871956bf6e975ef13b6954db1e7aR10).